Runtime/Managers/EditMediator.cs

```diff
@@ -74,6 +74,9 @@ public IChangeMaterialController changeMaterialController
         private IOriginalSceneController _originalSceneController;
         public IOriginalSceneController originalSceneController { get => _originalSceneController; set => _originalSceneController = value; }
 
+        private IBuildingVisibilityController _buildingVisibilityController;
+        public IBuildingVisibilityController buildingVisibilityController { get => _buildingVisibilityController; set => _buildingVisibilityController = value; }
```

Comment:
another perk I wish I had discovered ages ago
```suggestion
        public IBuildingVisibilityController buildingVisibilityController { get; set; }
```